databases/week2/Part3.sql

```diff
@@ -0,0 +1,27 @@
+use hyf_lesson2;
+
+-- 1
+select id, name, email
+from user
+WHERE email LIKE "%@spotify.com";
```

Comment:
I think you misread this exercise description here. It says to "Get all the **tasks** assigned to users whose email ends in @spotify.com" :)

Reply:
Oh my..., how could I missed this. Thank you.